dojo/templates/dojo/users.html

```diff
@@ -53,7 +53,7 @@ <h3 class="has-filters">
                             <th class="nowrap">Phone Number(s)</th>
                             <th class="nowrap, text-center">{% dojo_sort request 'Active' 'is_active' %}</th>
                             <th class="nowrap, text-center">{% dojo_sort request 'Staff' 'is_staff' %}</th>
```

Comment:
should we remove this also? notify that it will be deprecated soon?

Reply:
That's for the next release, there are some steps to do until then.

Reply:
ok